### PR TITLE
Bump to 1.7

### PR DESCRIPTION
**Pull request by brian-avery**
This PR bumps for 1.7

### CODE REVIEW

File: manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.7.yaml [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+  {{- end }}
+  labels:
+    istio.io/rev: {{ .Values.revision | default "default" }}
+spec:
+  configPatches:
+    - applyTo: HTTP_FILTER
+      match:
+        context: ANY # inbound, outbound, and gateway
+        proxy:
+          proxyVersion: '^1\.6.*'
```
Comment (howardjohn) [inline manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.7.yaml:19]: this should be 7

**Review by howardjohn — APPROVED** [review]
/hold to fix the v2

**Comment by howardjohn** [comment]
I guess we also need some translateConfig/names-1.7.yaml file

File: manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.7.yaml (hunk 1 above)
Comment (brian-avery) [reply manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.7.yaml]: fixed. Thanks.

**Comment by brian-avery** [comment]
Added translateConfig/names-1.7.yaml. Thanks.

**Pull request merged**